docs/overview.md

```diff
@@ -1,7 +1,8 @@
 ---
 name: SingleStore
-title: Fivetran for SingleStore | Configuration and Documentation
+title: SingleStore Destination connector by Fivetran | Fivetran documentation
```

Comment:
Why is it called "by Fivetran"? This seem to be inaccurate as the connector is buult by SingleStore.

Reply:
It was copied from their docs template.
But the template is already updated and the title format is changed.
I will create a separate ticket to update the docs.

Reply:
https://memsql.atlassian.net/browse/PLAT-7363